components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -217,17 +217,17 @@ class GeckoEngineSession(
     /**
      * See [EngineSession.goBack]
      */
-    override fun goBack() {
-        geckoSession.goBack()
+    override fun goBack(userInteraction: Boolean) {
+        geckoSession.goBack(userInteraction)
         if (canGoBack) {
             notifyObservers { onNavigateBack() }
```

Comment:
Is this something that we also wan to communicate to listeners of `onNavigateBack` ? :)

Reply:
For now, probably not. I'm not sure I can think of a reason to expose it to the EngineSession observer.

If we find a need for it, we can do that in a follow-up.

Reply:
👍🏽